<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="fileTypeLbl">
       <property name="text">
        <string>File Type:</string>
       </property>
       <property name="buddy">
        <cstring>fileType</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fileType">
       <item>
        <property name="text">
         <string>pdf</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>svg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpeg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tiff</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bmp</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="heightLbl">
          <property name="text">
           <string>Height:</string>
          </property>
          <property name="buddy">
           <cstring>height</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="height">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="widthLbl">
          <property name="text">
           <string>Width:</string>
          </property>
          <property name="buddy">
           <cstring>width</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="width">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="resolutionLbl">
          <property name="text">
           <string>Resolution:</string>
          </property>
          <property name="buddy">
           <cstring>resolution</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="resolution">
          <property name="minimum">
           <number>72</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="selectionLbl">
          <property name="text">
           <string>Selection:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="chartSelection">
          <item>
           <property name="text">
            <string>Current Chart</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>All Charts</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
